src/main/java/controller/Controller.java

```diff
@@ -0,0 +1,13 @@
+package controller;
+
+import model.HttpRequest;
+import model.HttpResponse;
+
+import java.io.IOException;
+import java.net.URISyntaxException;
+
+public interface Controller {
+    boolean matchHttpMethodAndPath(HttpRequest request);
```

Comment:
메서드명에 내부 로직을 상세히 적을 필요는 없어 보이네요.

```suggestion
    boolean match(HttpRequest request);
```